docs/guides/shell-completion.md

```diff
@@ -2,39 +2,42 @@
 title: Enable Completion Script
 ---
 
-# Enable shell completion
+# Enabling shell completions
 
-Below is example steps to enable shell completion feature for `bearer` cli:
+Completions make it easier to use bearer on the command line. In this guide we will cover how to set everything up.
 
-## 1. Know your current shell
+## Discover your current shell
 
 ```bash
 $ echo $SHELL
-/bin/zsh # For this example it is zsh, but will be vary depend on your $SHELL, maybe /bin/bash or /bin/fish
+/bin/zsh
 ```
+In our example it's zsh, but will be vary depend on your $SHELL.
 
-## 2. Run `completion` command to get sub-commands
+## Test the completion scripts
 
-``` bash
-bearer completion -h
-```
+We currently support `zsh` `bash` and `fish`. To load completions in your current shell session you can run the following:
 
-Generate the autocompletion script for the zsh shell.
+```bash
+source <(bearer completion zsh); compdef _bearer bearer
+```
 
-If shell completion is not already enabled in your environment you will need
-to enable it.  You can execute the following once:
+Now you can test completions:
 
 ```bash
-echo "autoload -U compinit; compinit" >> ~/.zshrc
+bearer [tab]
 ```
 
-To load completions in your current shell session:
+**Note:** if completions are not already enabled in your environment you will need
+to enable it. In zsh you can execute the following once:
 
 ```bash
-source <(bearer completion zsh); compdef _bearer bearer
+echo "autoload -U compinit; compinit" >> ~/.zshrc
 ```
 
-To load completions for every new session, execute once:
+## Final setup
+
+To load completions for every new session you can add the test command to your `~/.zshrc` or similar. Alternatively if you are using zsh you can execute the following once:
```

Comment:
It may be useful to repeat the test command here (sorry - suggestion markdown is getting confused with the "```" but you see the idea)

```suggestion
To load completions for every new session, you can add the test command to your `~/.zshrc` or similar: 

`bash
  source <(bearer completion zsh); compdef _bearer bearer
`

Alternatively, if you are using zsh, you can execute the following once:
```

Reply:
Think its overkill in this case.